dss-api.yml

```diff
@@ -2257,6 +2257,194 @@ paths:
+                    enum: [unhandled_exception, Forbidden, Unauthorized, OAuthResponseProblem, not_found]
+                required:
+                  - code
+  /events/{uuid}:
+    get:
+      summary: Retrieve a bundle metadata document given a UUID and version.
+      description: >
+        Given a bundle UUID and version, return the bundle metadata document.
+      parameters:
+        - name: uuid
+          in: path
+          description: Bundle unique ID.
+          required: true
+          type: string
+          pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+        - name: version
+          in: query
+          description: Timestamp of bundle creation in DSS_VERSION format.
+          required: true
+          type: string
+        - name: replica
+          in: query
+          description: Replica to fetch from.
+          required: true
+          type: string
+          enum: [aws, gcp]
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+        400:
+          description: Bad request
+          schema:
+            allOf:
+              - $ref: '#/definitions/Error'
+              - type: object
+                properties:
+                  code:
+                    type: string
+                    description: >
+                      Machine-readable error code.  The types of return values should not be changed lightly.
+
+
+                      The code `illegal_arguments` is returned when required parameters are missing or parameters do not
+                      match the requirements in the specification.
+                    enum: [illegal_arguments]
+                required:
+                  - code
+        500:
+          description: Server Error.
+          headers:
+            Retry-After:
+              description: Delay in seconds, service clients should retry after the delay.
+              type: integer
+              format: int64
+          schema:
+            allOf:
+              - $ref: '#/definitions/Error'
+              - type: object
+                properties:
+                  code:
+                    type: string
+                    description: >
+                      Machine-readable error code.  The types of return values should not be changed lightly.
+
+
+                      The code `unhandled_exception` is returned when an unexpected exception is encountered.
+                    enum: [unhandled_exception]
+                required:
+                  - code
+        502:
+          $ref: '#/responses/BadGateway'
+        503:
+          description: Service Unavailable.
+          headers:
+            Retry-After:
+              description: Delay in seconds, service clients should retry after the delay.
+              type: integer
+              format: int64
+          schema:
```

Comment:
Would it be useful to add this schema to all 503 responses by adding it here?  https://github.com/HumanCellAtlas/data-store/blob/master/dss-api.yml#L2730

Reply:
Perhaps, but in a separate PR.